prow/config.yaml

```diff
@@ -1821,6 +1821,41 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  jlewi/mlkube.io:
+    - name: mlkube-build
+      agent: kubernetes
+      context: mlkube-build
+      always_run: true
+      rerun_command: "/test mlkube-build"
+      trigger: "(?m)^/test( all| mlkube-build),?(\\s+|$)"
+      spec:
+        containers:
+        - image: gcr.io/mlkube-testing/builder:v20171010-352a232c-dirty
+          # TODO(jlewi): Add a service account to push to GCR?
+          # args:
+          env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/service-account/service-account.json
+          # TODO(jlewi): Do we need privileged mode in order to run Docker?
```

Comment:
you probably need to figure out https://github.com/kubernetes/kubernetes/issues/1806, have you tried locally?

Reply:
I've tried running my container locally using docker run but not running on K8s. Going to try it on my test prow cluster soon.